js/nav.js

```diff
@@ -44,7 +24,5 @@ handleMobileNav = function() {
 
 // Call Functions
 $(document).ready(function(){
-  handleMainNav();
-  handleMobileNav();
   handleInitialHide();
```

Comment:
Is there another way to handle calling this now we have controllers and all that?